tests/api2/test_190_filesystem.py

```diff
@@ -8,8 +8,9 @@
 import os
 apifolder = os.getcwd()
 sys.path.append(apifolder)
-from functions import POST
-from auto_config import dev_test
+from functions import POST, SSH_TEST
+from auto_config import dev_test, pool_name
+from config import BSD_HOST, BSD_USERNAME, BSD_PASSWORD
```

Comment:
I think you'll want:
```
from auto_config import dev_test, ip, pool_name, user, password
```

user, password here will be privileged so you'd do:

```
    results = SSH_TEST(cmd, user, password, ip)
```

But my memory may be fuzzy. Double-check with Eric.